tests/model_executor/test_guided_processors.py

```diff
@@ -16,7 +16,9 @@
 from vllm.sampling_params import GuidedDecodingParams
 
 MODEL_NAME = 'HuggingFaceH4/zephyr-7b-beta'
-GUIDED_DECODING_BACKENDS = ["outlines", "lm-format-enforcer", "xgrammar"]
+GUIDED_DECODING_BACKENDS = [
+    "outlines", "lm-format-enforcer", "xgrammar", "guidance"
```

Comment:
can you also add this to the entrypoints tests?

Reply:
good catch

Reply:
done

Reply:
hmm, seems like the tests failed with outlines due to non-deterministic

Reply:
yes - not related to this PR.

It (outlines) allowed an array of objects instead of just an object. I'd say that's a bug, but not related to this PR.